docs/source/models/gbd2017/causes/latent_tuberculosis/index.rst

```diff
@@ -4,6 +4,119 @@
 Latent Tuberculosis
 ===================
 
-.. todo:
+TB Overview
+-----------
 
-   all of this
+TB definitions
+++++++++++++++
+Tuberculosis (TB) is a disease caused by the bacteria Mycobacterium
+tuberculosis. It most commonly affects the lungs, although it can affect
+other parts of the body. TB can be fatal if not recognized and treated.
+It also can spread from person to person to infect others. Anyone who 
+inhales the airborne droplets that contains TB bacteria is called a "contact".
+A contact can be someone who living with the person who has infectious TB,
```

Comment:
is infectious TB==active TB? I think it's worth using consistent language there

Reply:
I think the infectious disease docs make a distinction between active TB that "pulmonary TB" (affecting the lungs) and therefore is infectious, and "extrapulmonary TB", which is active but unlikely to infect others.  However, we do not use this level of detail in our sim.  I support changing this phrase to "active TB" or striking infectious and writing just "TB"